data/net/minecraft/world/inventory/MerchantResultSlot.mapping

```diff
@@ -1,4 +1,11 @@
 CLASS net/minecraft/world/inventory/MerchantResultSlot
+	METHOD <init> (Lnet/minecraft/world/entity/player/Player;Lnet/minecraft/world/item/trading/Merchant;Lnet/minecraft/world/inventory/MerchantContainer;III)V
+		ARG 1 player
+		ARG 2 merchant
+		ARG 3 merchantContainer
```

Comment:
Point of 'parameters follow field names': `merchantContainer` -> `slots`. (Again, this doesn't sit well with me, but alas.)